src/tests/ftest/mpiio/llnl_mpi4py.py

```diff
@@ -46,8 +46,8 @@ def test_mpi4py(self):
         testIReadIWriteAll, testReadWriteAllBeginEnd
 
         :avocado: tags=all,daily_regression
-        :avocado: tags=large
-        :avocado: tags=mpiio,mpich,mpi4py
+        :avocado: tags=vm
+        :avocado: tags=mpiio,mpich,mpi4py,test_mpi4py
```

Comment:
Minor. Feature tags vs test tags
```suggestion
        :avocado: tags=mpiio,mpich,mpi4py
        :avocado: tags=test_mpi4py
```